Library/src/Multiplayer/EventSerialisation.cpp

```diff
@@ -71,6 +71,11 @@ csp::multiplayer::ReplicatedValue EventDeserialiser::ParseSignalRComponent(uint6
 {
 	csp::multiplayer::ReplicatedValue ReplicatedValue;
 
+	if (Component.type() == signalr::value_type::null)
```

Comment:
I had to add this line to the base class serializer to prevent as crash, due to the third event param being null unless the UpdateType == Update

Reply:
Worth including a comment in the code to this effect? Thinking of future us encountering this line and not understanding the context around why it is necessary,